Tests iOS/Fixtures/corpusSlates.json

```diff
@@ -89,6 +89,30 @@
                                 }
                             }
                         },
+                        {
+                            "__typename": "[some-type-name]",
+                            "id": "619b6058-49e5-40b8-af43-dcca6048e61e",
+                            "corpusItem": {
+                                "__typename": "[some-type-name]",
+                                "id": "item-4",
+                                "url": "https://getpocket.com/item/slate-2-rec-2",
+                                "title": "Syndicated Article Slate 2, Rec 2",
+                                "excerpt": "Cursus Aenean Elit",
+                                "imageUrl": "http://example.com/slate-2-rec-2/top-image.png",
+                                "publisher":"slate-2-rec-1.example.com",
+                                "target": {
+                                    "__typename": "SyndicatedArticle",
+                                    "itemId": "slate-2-rec-2-corpusItem-4-syndicated",
+                                    "mainImage": "image-2-2-4",
+                                    "title": "syndicatedTitle-2-2-4",
+                                    "excerpt": "syndicatedExcerpt-2-2-4",
+                                    "publisher": {
+                                      "__typename": "Publisher",
+                                      "name": "syndicatedPublisher-2-2-4"
+                                    }
+                                }
+                            }
+                        },
```

Comment:
Hey @cyndichin , instead of adding another item in this fixture, would it be possible to use `item-2`, which is already `SyndicatedArticle` for this purpose? Reason I am asking is because I am adding another item to this same fixture, which needs not to be a `SyndicatedArticle`, and would try to keep the fixture as compact as possible. What do you think?

Reply:
I wanted to mimic our old `slates.json` file to avoid breaking more tests. A couple of tests rely on `Syndicated Article Slate 2, Rec 2` which is missing from the `corpusSlates.json`. If we want to test an item that is not a `SyndicatedArticle`, shall we use Slate 1, Rec 1 for that purpose?

Overall, would it be beneficial to modify the fixtures so that they are similar as opposed to having to modify test code that may span across different classes?

Reply:
> shall we use Slate 1, Rec 1 for that purpose?

that one is a collection, and not sure if it's used that way in other tests

Reply:
> A couple of tests rely on Syndicated Article Slate 2, Rec 2

if we know what they are, can we just update the reference in those tests too? Still with the purpose of not making the fixture too big

Reply:
`Slate 1, Rec 2 and Slate 2, Rec 1` are also not syndicated in the old `slates.json`, so those are valid as well

The other ones are in `HomeTests`, so we can update them when working on that ticket. However, if I modify the other recommendation items to be syndicated, it may impact the previous tests that rely on the recommendations not being syndicated articles.

Reply:
happy to go either direction, but just wanted to express those concerns

Reply:
What I am thinking is:
- to fix `ReportARecommendationTests`, update the references in it, so it uses `item-2` or `item-3`. This would be the least amount of changes, with no changes in fixtures: this should ensure that we do not introduce other failures
- I am working on `HomeTests` and will fix anything that need fixing in there
Does that work?

Reply:
Thanks! I updated the tests and decided to use `item-2`. I had to add a new fixture to replace `recommendation-detail-4` so that we don't introduce any new failures.